### PR TITLE
Export `DiscoveredService` class

### LINKED ISSUE #125: Class DiscoveredService isn't exported
I'm trying to create a function that discovers services for a deviceId and returns the result. It's really just a small change from the function in the example project.
I'm changing
`Future<void> discoverServices(String deviceId) async {
    ...
}`
to
`Future<List<DiscoveredService>> discoverServices(String deviceId) async {
    ...
}`

I'm having troubles importing/using the type `DiscoveredService` since the file `discovered_services.dart` isn't exported in `flutter_reactive_ble.dart` in the library.

Is this intentional or could we add `discovered_services.dart` to the list of exported files in `flutter_reactive_ble.dart`? If so I'm happy to make a PR with that small change.

### PR DESCRIPTION
**Pull request by werediver**
Fixes #125

### CODE REVIEW

**Review by remonh87 — APPROVED** [review]

File: lib/src/converter/protobuf_converter.dart [hunk 1]
```diff
@@ -7,7 +7,7 @@ import 'package:flutter_reactive_ble/src/model/clear_gatt_cache_error.dart';
 import 'package:flutter_reactive_ble/src/model/connection_priority.dart';
 import 'package:flutter_reactive_ble/src/model/connection_state_update.dart';
 import 'package:flutter_reactive_ble/src/model/discovered_device.dart';
-import 'package:flutter_reactive_ble/src/model/discovered_services.dart';
+import 'package:flutter_reactive_ble/src/model/discovered_service.dart';
```
Comment (remonh87) [inline lib/src/converter/protobuf_converter.dart:10]: just a small s maybe something to consider for us to revise to avoid confusion

Comment (werediver) [reply lib/src/converter/protobuf_converter.dart:10]: Likely we had some different classes (or different naming) there before we settled on the current implementation.

File: lib/src/model/discovered_service.dart [hunk 2]
```diff
@@ -22,5 +22,3 @@ class DiscoveredService extends $DiscoveredService {
   @CustomEquality(DeepCollectionEquality())
   final List<DiscoveredService> includedServices;
 }
-
-enum DiscoverServicesFailure { unknown }
```
Comment (werediver) [inline lib/src/model/discovered_service.dart:26]: This is also a leftover.

**Comment by werediver** [comment]
Should be good now, but I was working with an older version of Flutter SDK.

**Pull request merged**